addons/cookoff/functions/fnc_handleDamage.sqf

```diff
@@ -16,16 +16,16 @@
 #include "script_component.hpp"
 
 params ["_simulationType", "_thisHandleDamage"];
-_thisHandleDamage params ["_vehicle", "", "_damage", "_source", "_ammo", "_hitIndex", "_shooter"];
+_thisHandleDamage params ["_vehicle", "", "_damage", "_source", "_ammo", "_hitIndex", "_shooter", "_hitPoint"];
 
 // it's already dead, who cares?
 if (damage _vehicle >= 1) exitWith {};
 
-// get hitpoint name
+// set default value
 private _hitpoint = "#structural";
 
 if (_hitIndex != -1) then {
-    _hitpoint = toLower ((getAllHitPointsDamage _vehicle param [0, []]) select _hitIndex);
+    _hitpoint = toLower (_hitPoint);
```

Comment:
but vars are casesensitive. arent they?
Sure it could be clarified with using another var that _hitPoint in the param.
params use _hitPoint the rest is _hitpoint.